s3/src/main/scala/akka/stream/alpakka/s3/javadsl/S3Client.scala

```diff
@@ -56,9 +56,14 @@ final class S3Client(s3Settings: S3Settings, system: ActorSystem, mat: Materiali
     impl.download(S3Location(bucket, key), Some(scalaRange)).asJava
   }
 
-  // #list-bucket
```

Comment:
travis will do 'sbt paradox' to generate the documentation, and the documentation still refers to this snippet (marked by `// #list-bucket`)